### DISCUSSION TITLE
Instructions for contributing to rules?

### DISCUSSION THREAD

**Discussion by leogama** [OhMyGuus/I-Still-Dont-Care-About-Cookies#13474] [Q&A]
Hello. I'm from Brazil and interested in contributing new rules for Brazilian websites. New legislation entered in effect recently and now there are cookie banners absolutely everywhere. I can see that the syntax in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/blob/master/src/data/rules.js is pretty straightforward, but **I don't understand the file organization** (if there's any) and how exactly the handlers work.

Are there instructions for contributing anywhere?

**Comment by erik55** [comment]
I couldn’t find instructions. But you can find a `tools` folder in the project. There is a README file which has some examples. Mostly it works with a css rule that hides the annoying cookie banner. Sometimes you have to use a click handler rule or something. For that you can check examples in the `src/data/js/5_clickHandler.js` file.
